sdk/eventhub/azure-eventhubs/azure/eventhub/aio/client_async.py

```diff
@@ -85,33 +97,31 @@ def _create_auth(self, username=None, password=None):
                                                 get_jwt_token, http_proxy=http_proxy,
                                                 transport_type=transport_type)
 
+    async def _handle_exception(self, exception, retry_count, max_retries):
+        await _handle_exception(exception, retry_count, max_retries, self, log)
```

Comment:
I don't think we should be passing `log` here - there should be a logger defined in the error handler module

Reply:
fixed